### PR TITLE
Added a new formula for the ec2backup tool (manages EC2 volumes snaps…

### PR DESCRIPTION
**Pull request by tanis2000**
…hots)

### CODE REVIEW

File: Formula/ec2backup.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Ec2backup < Formula
+  desc "Simple AWS EC2 backup command line application for MacOS, Windows and Linux"
```
Comment (MikeMcQuaid) [inline Formula/ec2backup.rb]: `"Simple AWS EC2 backup application"

File: Formula/ec2backup.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  def install
+    ENV["GOPATH"] = buildpath.to_s
+    mkdir_p buildpath/"src/github.com/tanis2000"
+    ln_s buildpath, buildpath/"src/github.com/tanis2000/ec2backup"
+    system "go", "install", "github.com/tanis2000/ec2backup"
+    bin.install "#{buildpath}/bin/ec2backup"
+  end
+
+  test do
+    system "#{bin}/ec2backup", "--version"
```
Comment (MikeMcQuaid) [inline Formula/ec2backup.rb]: Use `assert_match` and `shell_output` to verify the output.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
Passing on this as we don't generally accept self-submissions of formulae, sorry. Please consider [creating your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) instead. Thanks!

**Comment by tanis2000** [comment]
@MikeMcQuaid sure. I noticed the automatic build tool said that there were not enough forks and stars to be accepted. No worries.

**Pull request closed without merging**